sql-to-dbsp-compiler/lib/sqllib/src/timestamp.rs

```diff
@@ -602,7 +597,7 @@ impl Serialize for Time {
         S: Serializer,
     {
         let time = self.to_time();
-        serializer.serialize_str(&time.format("%H:%M:%S.9f").to_string())
+        serializer.serialize_str(&time.format("%H:%M:%S.%9f").to_string())
```

Comment:
how come the tests passed?

Reply:
No idea, maybe you are not using this type?

Reply:
maybe you can fix the documentation too?

Reply:
I think this fix is wrong, it should be %.9f

Reply:
Well we can fix it in your PR hopefully: https://github.com/feldera/dbsp/pull/440

Reply:
https://docs.rs/chrono/latest/chrono/naive/struct.NaiveTime.html#method.format